extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -53,6 +56,70 @@ export class LocalSubscriptionPlatform {
         ),
         'Authorization Url'
     );
+
+    // TODO(dvoytenko, #3742): This will refer to the ampdoc once AccessService
+    // is migrated to ampdoc as well.
+    /** @private @const {!Promise<!../../../src/service/cid-impl.Cid>} */
+    this.cid_ = Services.cidForDoc(ampdoc);
+
+    /** @private {UrlBuilder} */
```

Comment:
All nullable should be `?Xyz`. But, why not just initialize them right away. There's nothing to wait for, right?

Reply:
Done